packages/patternfly-4/react-integration/demo-app-ts/src/components/demos/ApplicationLauncherDemo/ApplicationLauncherFavoritesDemo.tsx

```diff
@@ -44,7 +44,7 @@ export class ApplicationLauncherFavoritesDemo extends React.Component {
       isOpen
     });
   };
-  onSelect = event => {
+  onSelect = () => {
```

Comment:
As a demo, it would be good to show what args can be are expected here

Reply:
Added this back in